packages/matter.js/tsconfig.base.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "target": "ES2015",
     "module": "ES2015",
+    "strictNullChecks": true,
```

Comment:
Same, add comment

Reply:
added  // Required to correctly generate Nullable types in d.ts